controllers/compassmanager_controller.go

```diff
@@ -428,3 +306,198 @@ func createCompassRuntimeLabels(kymaLabels map[string]string) map[string]interfa
 
 	return runtimeLabels
 }
+
+type ClusterInterface struct {
```

Comment:
Compass Manager will have two `cluster` contexts. One for `control-plane` and the other for `SKR`. The name should make it clear what context is being referred to. In my opinion, the keyword `cluster` should be referred to `SKR`. We should change the naming to something more appropriate.

Here it is my proposals:
 - ControlPlaneInterface
 - CentralInterface
 - InfrastructureInterface

What do you think about it?

Reply:
`ControlPlaneInterface` is my favourite out of those.
Or maybe `KCPInterface`?

Reply:
KCPInterface is good